pom.xml

```diff
@@ -2880,11 +2893,15 @@
         <version>${maven-site.version}</version>
         <configuration>
           <siteDirectory>${basedir}/src/site</siteDirectory>
-          <customBundle>${basedir}/src/site/custom/project-info-report.properties</customBundle>
```

Comment:
There was a warning regarding this during the build:

```
[WARNING] Parameter 'customBundle' is unknown for plugin 'maven-site-plugin:3.12.0:site (default-site)'
```

Removing this line fixed it.

Reply:
I checked the pages they seem to be good to me.
But to be on the safe side we can also revert this file back.

Reply:
I've looked into this:

That bundle is used by the reports plugin, and the properties file is in the default location.
So it looks like the config was just in the wrong plugin.

I suggest keeping the bundle file, but removing the incorrect config from the site plugin.

https://maven.apache.org/plugins/maven-project-info-reports-plugin/team-mojo.html

Reply:
Many thanks for looking more into this. Nice find. :+1: 

I'll restore the file then. :)

Reply:
Thanks, good idea. :+1:  OK, I tried to change some string and some was reflected on the site but some of them was not.
I checked the bundle file in jar file of the reports plugin version we use and some keys are different. I updated our file and now the texts are displayed on the pages.